Clappr-tvOS/Classes/Formatter/ClapprDateFormatter.swift

```diff
@@ -0,0 +1,11 @@
+open class ClapprDateFormatter {
+    fileprivate static let hourInSeconds: Double = 1 * 60 * 60
```

Comment:
This class could be easily merged into the shared one.

Reply:
Done!